pkg/streamingpromql/operators/aggregation.go

```diff
@@ -276,14 +288,20 @@ func (a *Aggregation) accumulateUntilGroupComplete(ctx context.Context, g *group
 
 		thisSeriesGroup := a.remainingInnerSeriesToGroup[0]
 		a.remainingInnerSeriesToGroup = a.remainingInnerSeriesToGroup[1:]
-		err = a.accumulateSeriesIntoGroup(s, thisSeriesGroup, a.Steps, a.Start, a.Interval)
-		if err != nil {
+		if err := a.accumulateSeriesIntoGroup(s, thisSeriesGroup, a.Steps, a.Start, a.Interval); err != nil {
 			return err
 		}
+
+		a.currentSeriesIndex++
 	}
 	return nil
 }
 
+// Sentinel value used to indicate a sample has seen an invalid combination of histograms and should be ignored.
+//
+// Invalid combinations include exponential and custom buckets, and histograms with incompatible custom buckets.
+var invalidCombinationOfHistograms = &histogram.FloatHistogram{}
```

Comment:
Why use a global here instead of on the struct? I don't think the memory overhead would be significant

Reply:
I would make this a const if that was valid in Go.

It doesn't make sense to me to add it to the struct, as there's nothing about it that's specific to this aggregation instance.